doc/tutorials/objdetect/traincascade.markdown

```diff
@@ -287,13 +287,18 @@ Command line arguments of opencv_traincascade application grouped by purposes:
 
     -   -minHitRate \<min_hit_rate\>
 
-        Minimal desired hit rate for each stage of the classifier. Overall hit rate may be estimated
-        as (min_hit_rate\^number_of_stages).
+        Minimal desired hit rate for each stage of the classifier. The overal hit rate can be calculated as follows.
+
+        overall_hit_rate = 1.0 - overall_fail_rate
+        overall_fail_rate  = max_fail_rate ^ number_of_stages
+        max_fail_rate = 1.0 - min_hit_rate
+
+        and thus overall_hit_rate = ( 1.0 - ( 1.0 - min_hit_rate ) ^ number_of_stages )
```

Comment:
Actually I guess you should read up on the discussion. It is quite strange to see that you are making a classifier that has a worse hit rate after each stage. For example in this case, if your overall hit rate would be `0.99^20 = 0.81` then that would mean that adding more stages actually makes your detector worse, a lot worse... which seems onlogical to me and many others. If you look at the reasoning of alalek, starting from the definition of hit rate in function of fail rate, this seems way more logical. Result for this would then be `(1.0 - (1.0 - 0.99)^20) = something very close to 1` which is what you are aiming for, keeping an as high possible hit rate on the positive training set.